tests/static/js/jqunit/jqUnit-binder-commonEventBindings.js

```diff
@@ -0,0 +1,158 @@
+                            args: ["#input-button"]
+                        }
+                    ]
+                },
+                {
+                    name: "Test markup event binding for keydown on an input button",
+                    type: "test",
+                    expect: 1,
+                    sequence: [
+                        {
+                            func: "gpii.tests.binder.keydownSelector",
+                            args: ["#input-button"]
+                        }
+                    ]
+                },
+                {
+                    name: "Test markup event binding for click on a paragraph with a class",
+                    type: "test",
+                    expect: 1,
+                    sequence: [
+                        {
+                            func: "gpii.tests.binder.clickSelector",
+                            args: [".paragraph-click"]
+                        }
+                    ]
+                },
+                {
+                    name: "Test markup event binding for both `click` and `keydown` on an input button",
+                    type: "test",
+                    expect: 2,
+                    sequence: [
+                        {
+                            func: "gpii.tests.binder.clickSelector",
+                            args: ["#input-multiple-events-button"]
+                        },
+                        {
+                            func: "gpii.tests.binder.keydownSelector",
+                            args: ["#input-multiple-events-button"]
+                        }
+                    ]
+                },
+                {
+                    name: "Test markup event binding for both `click` and `keydown` on an input button, each using a different handler",
+                    type: "test",
+                    expect: 2,
+                    sequence: [
+                        {
+                            func: "gpii.tests.binder.clickSelector",
+                            args: ["#input-multiple-handlers-events-button"]
+                        },
+                        {
+                            func: "gpii.tests.binder.keydownSelector",
+                            args: ["#input-multiple-handlers-events-button"]
+                        }
+                    ]
+                }
+            ]
+        }]
+    });
+
+    fluid.defaults("gpii.tests.binder.commonEventBindings.environment", {
+        gradeNames:       ["gpii.tests.binder.environment"],
+        markupFixture:    ".viewport-common-event-bindings",
+        binderGradeNames: ["gpii.tests.binder.commonEventBindings"],
+        moduleName:       "Testing common event bindings",
+        components: {
+            commonEventBindingsTests: {
+                type: "gpii.tests.binder.commonEventBindings.caseHolder"
+            }
+        }
+    });
+
+    gpii.tests.binder.commonEventBindings.environment();
```

Comment:
I guess it matters less on the browser side (I think?), but for consistency with the rest of the package, this should use `fluid.test.runTests` instead of calling the environment directly.